doc/source/parameters/cfd/mortar_control.rst

```diff
@@ -0,0 +1,31 @@
+==============
+Mortar Control
```

Comment:
The subsection is called mortar, so maybe this could just be called Mortar instead of mortar control? I'm not sure